pipeline-model-definition/src/test/resources/multipleProperties.groovy

```diff
@@ -36,7 +36,7 @@ pipeline {
     options {
         buildDiscarder(logRotator(numToKeepStr:'1'))
         disableConcurrentBuilds()
-        skipDefaultCheckout(true)
+        skipDefaultCheckout()
```

Comment:
That makes sense as long as the `skipDefaultCheckout()` step can only ever be called from inside the `options` block.   Could I call it in a library?